README.md

```diff
@@ -0,0 +1,346 @@
+    Display both the system clock time and the elapsed time.
+-   `/timestamps/?currentTime&time=HH:mm:ss`\
+    Display the current time on the Jenkins master.
+
+Supported query parameters:
+
+-   **time** (since 1.8)\
+    Display the system clock time. Accepts the [JDK SimpleDateFormat](http://docs.oracle.com/javase/6/docs/api/java/text/SimpleDateFormat.html) format.\
+    The time zone and locale of the Jenkins server will be used, unless they are configured with the `timeZone` and `locale` query parameters.
+-   **elapsed** (since 1.8)\
+    Display the elapsed time since the start of the build. Accepts the [commons-lang DurationFormatUtils](https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/time/DurationFormatUtils.html) format.
+-   **precision** (since 1.3.2)\
+    Display the elapsed time in seconds, with a certain number of places after the decimal point. Accepts a number of decimal places or values such as `seconds` and `milliseconds`.
+-   **appendLog** (since 1.8)\
+    Display the console log line after the timestamp.
+-   **startLine** (since 1.8)\
+    Display the timestamps starting from a certain line. Accepts a positive integer to start at that line, or a negative integer to start that many lines back from the end.
+-   **endLine** (since 1.8)\
+    Display the timestamps ending at a certain line. Accepts a positive integer to finish at that line, or a negative integer to finish that many lines back from the end.
+-   **timeZone** (since 1.8)\
+    Time zone used when displaying the system clock time. Accepts the [JDK TimeZone](http://docs.oracle.com/javase/6/docs/api/java/util/TimeZone.html) ID format.
+-   **locale** (since 1.8)\
+    Select the locale to use when displaying the system clock time. Accepts a locale in the format recognised by [commons-lang LocaleUtils.toLocale](https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/LocaleUtils.html#toLocale(java.lang.String)).
+-   **currentTime** (since 1.8.8)\
+    Display the current time on the Jenkins master instead of reading timestamps from the build.
+
+Reading the timestamps directly from the file system is not recommended, because the format may change.
+
+### Java API
+
+**Since Timestamper 1.8**
+
+Other plugins can add a [dependency](https://wiki.jenkins-ci.org/display/JENKINS/Dependencies+among+plugins) on the Timestamper plugin, and then use the `TimestamperAPI.read` method to retrieve the timestamps.\
+The `read` method accepts any query string that can be passed to the `/timestamps/` URL.
+
+```java
+String query = "time=HH:mm:ss";
+try (BufferedReader reader = TimestamperAPI.get().read(build, query)) {
+    // read timestamps here
+}
+```
+
+### Release History
+
+New releases may take a few hours to appear in the update center.
+
+#### 1.10 and newer
+
+See the [GitHub release page](https://github.com/jenkinsci/timestamper-plugin/releases).
+
+#### 1.9 (2019-02-07)
```

Comment:
Releases prior to 1.10 could go to CHANGELOG.md and only be linked from the readme.

Reply:
@zbynek Added CHANGELOG.md, and migrated release notes to there. Also reference releases page for new release notes going forward.